packages/java/endpoint/src/main/java/com/vaadin/hilla/signals/config/SignalsConfiguration.java

```diff
@@ -62,11 +63,12 @@ public SecureSignalsRegistry signalsRegistry() {
      *
      * @return SignalsHandler endpoint instance
      */
-    @ConditionalOnFeatureFlag("fullstackSignals")
+    // @ConditionalOnFeatureFlag("fullstackSignals")
```

Comment:
Should be removed rather than commented out

Reply:
Done.